src/modules/tests/RemediationsWizard.test.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { remediationWizardTestData } from './testData';
+import { RemediationWizard } from '../RemediationsModal/RemediationsWizard';
+import configureStore from 'redux-mock-store';
+import promiseMiddleware from 'redux-promise-middleware';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('../../store/actions/host-actions', () => {
+  const actions = jest.requireActual('../../store/actions/host-actions');
+  return {
+    __esModule: true,
+    ...actions,
+    fetchHostsById: () => ({
+      type: 'FETCH_SELECTED_HOSTS',
+      payload: () => Promise.resolve([]),
+    }),
+  };
+});
+
+jest.mock('../../api/index', () => {
+  const api = jest.requireActual('../../api/index');
+  const { remediationWizardTestData } = jest.requireActual('./testData');
+  return {
+    __esModule: true,
+    ...api,
+    getResolutionsBatch: () =>
+      Promise.resolve(remediationWizardTestData.issueResolutionsResponse),
+  };
+});
+
+jest.mock(
+  '@redhat-cloud-services/frontend-components-utilities/interceptors',
+  () => ({
+    __esModule: true,
+    useAxiosWithPlatformInterceptors: jest.fn(() => ({
+      get: () => {
+        let res = {
+          meta: {
+            count: 2,
+            total: 2,
+          },
+          data: [
```

Comment:
https://fakerjs.dev/guide/ ;)

Reply:
hmmmm, im gonna set up another PR to get the app to like 50%, ill play around with faker in that. I completely forgot about it

Reply:
Was really just a hint that it could be used for generating some random data.